hazelcast/src/test/java/com/hazelcast/map/InterceptorTest.java

```diff
@@ -382,37 +383,8 @@ String getLoadedValue() {
         }
     }
 
-    static class MapInterceptorAdaptor implements MapInterceptor {
-
-        @Override
-        public Object interceptGet(Object value) {
-            return value;
-        }
-
-        @Override
-        public void afterGet(Object value) {
-        }
-
-        @Override
-        public Object interceptPut(Object oldValue, Object newValue) {
-            return newValue;
-        }
-
-        @Override
-        public void afterPut(Object value) {
-        }
-
-        @Override
-        public Object interceptRemove(Object removedValue) {
-            return removedValue;
-        }
-
-        @Override
-        public void afterRemove(Object value) {
-        }
-    }
-
     public static class SimpleInterceptor extends MapInterceptorAdaptor {
+        private static final long serialVersionUID = 1L;
```

Comment:
In overloaded methods of these classes, you don't call super.methodName() but you do for others. Let's pick one and use it for all.